VERSION

```diff
@@ -1 +1 @@
-1.0.0
+1.8.2
```

Comment:
probably be 1.9.0 (but if we  have 1.9.0 in develop and on release bumped 1.9.1  can replace it, then okay)